environs/manual/sshprovisioner/init_test.go

```diff
@@ -23,32 +24,34 @@ var _ = gc.Suite(&initialisationSuite{})
 
 func (s *initialisationSuite) TestDetectSeries(c *gc.C) {
 	response := strings.Join([]string{
-		"edgy",
+		"ubuntu",
+		"6.10",
 		"armv4",
 		"MemTotal: 4096 kB",
 		"processor: 0",
 	}, "\n")
 	defer installFakeSSH(c, sshprovisioner.DetectionScript, response, 0)()
-	_, series, err := sshprovisioner.DetectSeriesAndHardwareCharacteristics("whatever")
+	_, base, err := sshprovisioner.DetectBaseAndHardwareCharacteristics("whatever")
 	c.Assert(err, jc.ErrorIsNil)
-	c.Assert(series, gc.Equals, "edgy")
+	c.Assert(base, gc.Equals, corebase.MustParseBaseFromString("ubuntu@6.10"))
 }
 
 func (s *initialisationSuite) TestDetectionError(c *gc.C) {
 	scriptResponse := strings.Join([]string{
-		"edgy",
+		"ubuntu",
+		"6.10",
 		"ppc64le",
 		"MemTotal: 4096 kB",
 		"processor: 0",
 	}, "\n")
 	// if the script fails for whatever reason, then checkProvisioned
 	// will return an error. stderr will be included in the error message.
 	defer installFakeSSH(c, sshprovisioner.DetectionScript, []string{scriptResponse, "oh noes"}, 33)()
-	_, _, err := sshprovisioner.DetectSeriesAndHardwareCharacteristics("hostname")
+	_, _, err := sshprovisioner.DetectBaseAndHardwareCharacteristics("hostname")
 	c.Assert(err, gc.ErrorMatches, "subprocess encountered error code 33 \\(oh noes\\)")
 	// if the script doesn't fail, stderr is simply ignored.
 	defer installFakeSSH(c, sshprovisioner.DetectionScript, []string{scriptResponse, "non-empty-stderr"}, 0)()
-	hc, _, err := sshprovisioner.DetectSeriesAndHardwareCharacteristics("hostname")
+	hc, _, err := sshprovisioner.DetectBaseAndHardwareCharacteristics("hostname")
```

Comment:
Can you update this test (or add a new one) including a check on the returned base please?